toolbox/hgfs/hgfs_other.go

```diff
@@ -1,3 +1,6 @@
+//go:build !linux
+// +build !linux
+
 /*
 Copyright (c) 2017 VMware, Inc. All Rights Reserved.
```

Comment:
Thanks @yashdesai93 , ltgm but see below the "Go headers" check failed. I think you just need to change to a range:
> 2017-2022